### PR TITLE
fix(core): added default unique id for multiinput

### PR DESCRIPTION
**Pull request by Smiranin**
#### Please provide a link to the associated issue.
Closes [#6159](https://github.com/SAP/fundamental-ngx/issues/6159)

#### Please provide a brief summary of this pull request.
Added generation of a unique id if field `inputId` doesn't set

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: libs/core/src/lib/multi-input/multi-input.component.ts [hunk 1]
```diff
@@ -309,6 +310,9 @@ export class MultiInputComponent implements
                 this.buildComponentCssClass();
             })
         );
+        if (!this.inputId) {
+            this.inputId = uuidv4();
```
Comment (N1XUS) [inline libs/core/src/lib/multi-input/multi-input.component.ts:314]: Why not use it as a default value on line 115?

Comment (dimamarksman) [reply libs/core/src/lib/multi-input/multi-input.component.ts:314]: let's move it on the line 115 as initial value

Comment (Smiranin) [reply libs/core/src/lib/multi-input/multi-input.component.ts:314]: @N1XUS @dimamarksman For performance. When we move value to 115 line, uuidv4 will invoke for every case.
On this line, uuidv4 invoke only when user doesn't pass inputId.

**Review by N1XUS — APPROVED** [review]

**Comment by kavya-b** [comment]
image
it still shows non-unique id warning for the [mobile](https://deploy-preview-6260--fundamental-ngx.netlify.app/#/core/multi-input#mobile) example, is that expected?

**Review by dimamarksman — APPROVED** [review]

**Review by platon-rov — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

**Review by Betrozov — APPROVED** [review]

**Pull request merged**